src/structures/Message.js

```diff
@@ -498,23 +498,26 @@ class Message extends Base {
     /**
      * Deletes a message from the chat
      * @param {?boolean} everyone If true and the message is sent by the current user or the user is an admin, will delete it for everyone in the chat.
+     * @param {?boolean} [clearMedia = true] If true, any associated media will also be deleted from a device.
      */
-    async delete(everyone) {
-        await this.client.pupPage.evaluate(async (msgId, everyone) => {
+    async delete(everyone, clearMedia = true) {
```

Comment:
Should ClearMedia be true by default? this will delete the file from the device ?

Reply:
Yes, `clearMedia` is `true` by default in WA 
internals, also it was `true` before this PR.
I just added a choice for the user.